test/trip_dispatch_test.rb

```diff
@@ -78,8 +74,55 @@ def build_test_dispatcher
     end
   end
 
-  # TODO: un-skip for Wave 2
-  xdescribe "drivers" do
+  # Wave 3: tests for request_trip method
+  describe "request_trip" do
+    it "returns instance of trip" do
+      td = RideShare::TripDispatcher.new
+      expect(td.request_trip(1)).must_be_kind_of RideShare::Trip
+    end
+    # Were the trip lists for the driver and passenger updated?
+    it "Adds trip to passenger's trips" do
+      dispatcher = build_test_dispatcher
+      passenger = dispatcher.find_passenger(1)
+      dispatcher.request_trip(1)
+      expect(passenger.trips.length).must_equal 3
+    end
+
+    it "Adds trip to driver's trips" do
+      dispatcher = build_test_dispatcher
+      passenger = dispatcher.find_passenger(1)
+      dispatcher.request_trip(1)
+
+      # get the passenger's last trip
+      passengers_last_trip = passenger.trips[-1]
+      last_trips_driver = passengers_last_trip.driver_id
+      driver = dispatcher.find_driver(last_trips_driver)
+      expect(driver.trips.length).must_equal 5
+    end
+
+    it "Was the driver available" do
```

Comment:
this test name feels vague, and i'm unsure what we're testing for. Could it have a more clear name?